updating svg icons, updating styles for main event and adding readme …

pcsandford opened the pull request:
…text and images

@stulees @donnapep please review. I have added in the new icons as well as the text and images for the readme. I have also amended the styles to accommodate larger event titles and fixed some spacing issues.

I amended a one line in both current and next.js files to display the same text for consistency as well as adding a class to the main event display so I could style it.

Thanks!

stulees commented on README.md:
```diff
@@ -1,2 +1,130 @@
+
+## Steps to run the Current Event theme
+
+1. Fork or download the entire content-event-current-day repository. You will also need to run `npm install` and `bower install` to install the dependencies required for the the template. Bower is a package manager for Javascript libraries and frameworks. All third-party Javascript dependencies are listed in the bower.json file. To install Bower, run the following command in Terminal: `npm instal ...
```
@pcsandford _dependencies required for the the template_ should read _dependencies required for the the theme_

stulees commented on README.md:
````diff
@@ -1,2 +1,130 @@
+The next step is to locate the following code in the index.html file:
+
+```
+  <rise-google-calendar
````
@pcsandford This is old code from `index.html` regarding the use of `rise-google-calendar`. Need to update to what is current.

pcsandford replied:
@stulees sorry, I am not sure which part of this needs changing. The method for setting up the calendar was the one I used this time, or do you mean the html part?

stulees commented on README.md:
```diff
@@ -1,2 +1,130 @@
+
+The `calendar-id` attribute should be set to the id for your own public calendar. This id can be found by selecting calendar settings, clicking on the calendars link in the top line, and then clicking on the 'Edit settings' or 'Shared: Edit settings' link corresponding to the calendar you wish to collect the data from. The ID for the calendar can then be copied from here:![screenshot showing cale ...
+
+The start date and end date can be used to specify a specific range of dates. This theme looks for all upcoming events for the current day. 
```
@pcsandford I would remove the mention of start date and end date. These are not set on the instance in `index.html` and start date gets dynamically set in code according to the current day.

stulees commented on src/js/current.js:
```diff
@@ -12,7 +12,7 @@ RiseVision.ThemeEvents.Current = function() {
   function _noEvent() {
     // display message as event name
-    _$current.find(".main--eventName").text("No Event Scheduled");
+    _$current.find(".main--eventName").append("<p class = 'noEventText'>No Event Scheduled<p>");
```
@pcsandford Need to avoid creating HTML elements in the JS. A `<span>` is acceptable but anything else should either be directly in the HTML or use a `<template>`. This line should be changed to:

```
_$current.find(".main--eventName p").text("No Event Scheduled");
```

and put the `<p class='noEventText'></p>` in the HTML

donnapep commented:
@pcsandford If `calendar.svg` and `clock.svg` are being removed, do you think it makes more sense to rename `calendar2.svg` and `clock2.svg` to just `calendar.svg` and `clock.svg`?

pcsandford commented:
@donnapep @stulees thanks, I will make those changes. This is where I always go wrong - what is the correct method for doing this on here and in the command line? Do I close this PR and submit another one or do I submit another one in addition to this one with the changes set out above? Thanks!

donnapep commented:
@pcsandford You generally would want to make the changes in the same PR. Thx.

pcsandford commented:
@donnapep thanks! So once the changes are made I would commit and push them in the same manner as before?

donnapep commented:
@pcsandford Yes, that's correct. Then once you're given a thumbs up, you may want to squash your commits. Stu and I do that but I don't think it's critical.

pcsandford commented:
@donnapep thanks! I think last time I tried to squash 3 commits I ended up with 12 so I will probably practice that for next time :) Although now I know VIM a little better it might not be so bad.

pcsandford commented:
@donnapep @stulees please review. I have corrected the issues identified in the last commit and have added in the metadata and screenshot for the theme. Also corrected the google fonts url to https so it will work is storage. Thanks!

stulees commented:
@pcsandford Looks good :+1:

donnapep commented:
@pcsandford :+1:

pcsandford commented:
@donnapep @stulees thanks!
